templates/cli/lib/client.js.twig

```diff
@@ -154,7 +154,11 @@ class Client {
 
     let cookies = response.headers.getSetCookie();
     if (cookies && cookies.length > 0) {
-      globalConfig.setCookie(cookies[0]);
+      let cookieString = "";
+      cookies.forEach((cookie) => {
+          cookieString += cookie.split(";")[0] + ";";
```

Comment:
Why do we need to do this split?

Reply:
I think this makes sense, if first cookie is set to something else, doing this will save all the kookies.

Reply:
```suggestion
      globalConfig.setCookie(cookies.join(";"));
```
Would this do the job? 🤔

Reply:
That sounds simpler :). Thanks @Meldiron. I'll update the PR.